doc/tutorials/using_cat_on_up2.rst

```diff
@@ -78,38 +77,49 @@ for the HV shell (refer to :ref:`getting-started-up2` for setup instructions).
 
    .. code-block:: none
 
-      ACRN:\>wrmsr  -p1 0xd10  0xc0
-      ACRN:\>wrmsr  -p1 0xd11  0x3f
+      ACRN:\>wrmsr -p1 0xd10 0xc0
+      ACRN:\>wrmsr -p1 0xd11 0x3f
 
 #. Attach COS1 to PCPU1. Because MSR IA32_PQR_ASSOC [bit 63:32], we'll write 0x100000000
    to it to use CLOS1
 
    .. code-block:: none
 
-      ACRN:\>wrmsr   -p1   0xc8f    0x100000000
+      ACRN:\>wrmsr -p1 0xc8f 0x100000000
 
 .. _configure_cat_vm:
 
 Configure CAT for VM with VM Configuration
 ##########################################
 
-#. CAT on ACRN can be enabled and configured by modifying source code, so the first
-   step is to clone the ACRN source code (if you haven't already):
+#. CAT on ACRN can be enabled and configured by modifying source code. The first
+   step is to clone the ACRN source code (if you haven't done it already):
 
    .. code-block:: none
 
       $ git clone https://github.com/projectacrn/acrn-hypervisor.git
       $ cd acrn-hypervisor/
 
+#. Turn on CONFIG_CAT_ENABLED with ``make menuconfig`` command:
+
+   .. code-block:: none
+
+     $ cd hypervisor
```

Comment:
So I think a problem here may be that it will generate a `.config` file under `hypervisor/build/`, where if the user later tries to build ACRN from the top-level folder (which is what our "Building ACRN from Source" guide does), it will then not use it at all (as it would use `build/hypervisor/.config` if it existed)

Reply:
Played a bit and found that if the make command under acrn-hypervisor doesn't include "BOARD=", then it uses hypervisor/build/.config; but if the build from top-level with "BOARD=another_board", then it uses build/hypervisor/.config.

This is different from what is described in acrn-hypervisor/Makefile. Not sure if it's a bug?

#BOARD and SCENARIO definition priority:
#  If we do menuconfig in advance, the menuconfig will define
#     BOARD
#     SCENARIO
#  else if we have board/scenario file avaiable, BOARD and SCENARIO will be
#     extracted from files.
#  else if make comand has BORAD/SCENARIO parameters, BOARD and SCENARIO will
#     be gotten from parameters
#  else
#     default value defined in this make file will be used
#
 
In this tutorial, we may do similar to other tutorial to add the defconfig command. This is to imply that the user selects board here, then don't need to build it with "BOARD=" parameter. If it's needed, we may refine the "Building ACRN from Source".

$ make defconfig BOARD=apl-up2
$ make menuconfig

Reply:
Played a bit and found that if the make command under acrn-hypervisor doesn't include "BOARD=", then it uses hypervisor/build/.config; but if the build from top-level with "BOARD=another_board", then it uses build/hypervisor/.config.

This is different from what is described in acrn-hypervisor/Makefile. Not sure if it's a bug?

BOARD and SCENARIO definition priority:

If we do menuconfig in advance, the menuconfig will define
  BOARD
  SCENARIO
else if we have board/scenario file avaiable, BOARD and SCENARIO will be extracted from files.
else if make comand has BORAD/SCENARIO parameters, BOARD and SCENARIO will be gotten from parameters
else
default value defined in this make file will be used.

In this tutorial, we may do similar to other tutorial to add the defconfig command. This is to imply that the user selects board here, then don't need to build it with "BOARD=" parameter. If it's needed, we may refine the "Building ACRN from Source".

$ make defconfig BOARD=apl-up2
$ make menuconfig

Reply:
Hi @ZideChen0 , I was playing with `make` yesterday and also observed things that seems odd to me. Specifically, if I run `make all BOARD=nuc7i7dnb SCENARIO=industry` from the top-level `acrn-hypervisor` folder, it does not take the `BOARD` and `SCENARIO` parameter into account... it says at the very end that it compiled ACRN for `apl-mrb` and `sdc`... not sure what's happening and I haven't had time to investigate yet.